backend/src/main/java/ai/verta/modeldb/metadata/MetadataServiceImpl.java

```diff
@@ -280,4 +281,22 @@ public void deleteKeyValueProperties(
           responseObserver, e, DeleteKeyValuePropertiesRequest.Response.getDefaultInstance());
     }
   }
+
+  @Override
+  public void generateRandomName(
+      GenerateRandomNameRequest request,
+      StreamObserver<GenerateRandomNameRequest.Response> responseObserver) {
+    QPSCountResource.inc();
+    try (RequestLatencyResource latencyResource =
+        new RequestLatencyResource(ModelDBAuthInterceptor.METHOD_NAME.get())) {
+      responseObserver.onNext(
+          GenerateRandomNameRequest.Response.newBuilder()
+              .setName(Nomen.est().adjective().color().animal().get())
```

Comment:
```suggestion
              .setName(Nomen.est().adjective().color().animal().withSeparator("-").get())
```
@ad-47 this will use `-` as the separator instead of `_`, which will look more like the docker conventions.

Reply:
Resolved